stdlib/ref.mli

```diff
@@ -0,0 +1,49 @@
+(**************************************************************************)
+(*                                                                        *)
+(*                                 OCaml                                  *)
+(*                                                                        *)
+(*                        Nicolas Ojeda Bar, LexiFi                       *)
+(*                                                                        *)
+(*   Copyright 2020 Institut National de Recherche en Informatique et     *)
+(*     en Automatique.                                                    *)
+(*                                                                        *)
+(*   All rights reserved.  This file is distributed under the terms of    *)
+(*   the GNU Lesser General Public License version 2.1, with the          *)
+(*   special exception on linking described in the file LICENSE.          *)
+(*                                                                        *)
+(**************************************************************************)
+
+(** References.
+
+    References are mutable indirection cells containing a value of a given type.
+
+    @since 4.12 *)
+
+type 'a t = 'a ref = {mutable contents : 'a}
+(** The type of references of type ['a]. *)
+
+val ref : 'a -> 'a t
```

Comment:
I think this should be `create`. Note that we have `Atomic.make` though I personally think it's a mistake that should be corrected before the module goes into the world. (There should be a discussion with @gasche about that somewhere but I fail to find it on the `Atomic` module PR).